project_testing/housing_characteristics/Water Heater Efficiency.tsv

```diff
@@ -1,3 +1,3 @@
-Option=Electric Heat Pump, 80 gal	Option=Electric Premium	Option=Electric Standard	Option=Electric Tankless	Option=FIXME Fuel Oil Indirect	Option=Fuel Oil Premium	Option=Fuel Oil Standard	Option=Natural Gas Premium	Option=Natural Gas Standard	Option=Natural Gas Tankless	Option=Other Fuel	Option=Propane Premium	Option=Propane Standard	Option=Propane Tankless
-0.071429	0.071429	0.071429	0.071429	0.071429	0.071429	0.071429	0.071429	0.071429	0.071429	0.071429	0.071429	0.071429	0.071429
-# Created by: sources\recs\2009\tsv_maker.py
\ No newline at end of file
+"Option=Electric Heat Pump, 80 gal"	Option=Electric Premium	Option=Electric Standard	Option=Electric Tankless	Option=FIXME Fuel Oil Indirect	Option=Fuel Oil Premium	Option=Fuel Oil Standard	Option=Natural Gas Premium	Option=Natural Gas Standard	Option=Natural Gas Tankless	Option=Other Fuel	Option=Propane Premium	Option=Propane Standard	Option=Propane Tankless	"Option=Electric Standard, Scheduled"	"Option=Natural Gas Standard, Scheduled"	"Option=Electric Heat Pump, 50 gal, Scheduled"	"Option=Electric Heat Pump, 80 gal, Scheduled"
+0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556	0.055556
```

Comment:
Are these just being added for testing and not project_national?

Reply:
@ejhw All of the scheduled options for WH control are being added in so people could potentially look at them as upgrades, but national penetration is going to be set to zero. The intent with this branch is to add in a bunch of new functionality we could use, but not leverage any of it just yet until we have a better idea of national penetration of some of these things (if they are indeed even nonzero).